### ISSUE TITLE
Consider removing `annotation-description` and making that the default "purpose"

### ISSUE THREAD

**Issue by BigBlueHat** [w3c/annotation-aria#6]
From @aleventhal:
> rather than add an extra role here, how about the author simply not use any of the other annotation roles.

**Comment by BigBlueHat** [comment]
@aleventhal @TzviyaSiegman and I spoke today about `annotation-description` being essentially the default purpose of any thing pointed to by `aria-details` (i.e. the default for any annotation body). Consequently, it would be redundant to state that twice.

```html
<img src="cat.jpg" aria-details="cat-photo-info" alt="cat photo" />
<div id="cat-photo-info">
  A tabby colored short haired cat sleeping on a pillow.
</div>
```

**Comment by BigBlueHat** [comment]
Closed via 10cceee41331c3bd034e7eda24c92136af6cb1d8
